.github/ISSUE_TEMPLATE/1.bug.yml

```
@@ -34,17 +34,19 @@ body:
     attributes:
       label: Steps to reproduce 🕹
       description: |
-        Provide a link to a live example (you can use codesandbox.io) and an unambiguous set of steps to reproduce this bug.
-        Include code to reproduce, if relevant (which it most likely is).
+        Please provide a link to a live example and an unambiguous set of steps to reproduce this bug.
+             
+        As a starting point, we recommend you browse our [documentation](https://mui.com/x/advanced-components/), and [select](https://mui.com/static/x/forking-an-example.png) the closest example to your use case.
```

Maybe we should make this a general rule? So first do the change in http://github.com/mui/material-ui, add t the screenshot and then come back here?

```suggestion
        As a starting point, we recommend you browse our [documentation](https://mui.com/x/advanced-components/), and [select](https://mui.com/static/core/forking-an-example.png) the closest example to your use case.
```